Week1/app/src/main/java/com/example/week1/db/TodoViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package com.example.week1.db
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+
+class TodoViewModel(
+    todoDao: TodoDao
+) : ViewModel() {
+
+    private val repository: TodoRepository = TodoRepository(todoDao)
+    var currentTodoList: LiveData<List<Todo>> = repository.todoList
+
+    fun insertTodo(todo: Todo) {
+        viewModelScope.launch {
+            repository.insert(todo)
+        }
+    }
+
+    fun updateTodo(todo: Todo) {
+        viewModelScope.launch {
+            repository.update(todo)
+        }
+        currentTodoList = repository.todoList
+    }
+
+    fun deleteTodo(todo: Todo) {
+        viewModelScope.launch {
+            repository.delete(todo)
+        }
+        currentTodoList = repository.todoList
```

Comment:
```suggestion
    }

    fun deleteTodo(todo: Todo) {
        viewModelScope.launch {
            repository.delete(todo)
        }
```

livedata를 굳이 수정할 필요는 없을 것 같습니다.